api/base/middleware.py

```diff
@@ -1,5 +1,6 @@
-from framework.transactions import commands, messages, utils
 from pymongo.errors import OperationFailure
+from raven.contrib.django.raven_compat.models import sentry_exception_handler
+from framework.transactions import commands, messages, utils
```

Comment:
Needs a blank line before the `framework` import.

ref: http://cosdev.readthedocs.org/en/latest/style_guides/python.html#imports